### PR TITLE
Removed unused fonts

### PR DESCRIPTION
**Pull request by Intelligent2013**

### CODE REVIEW

File: .github/workflows/debian.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
 
       - name: Install fonts
         run: |
-          apt-get -y install fonts-freefont-ttf fonts-droid-fallback
+          apt-get -y install fonts-freefont-ttf
```
Comment (ronaldtse) [inline .github/workflows/debian.yml]: Should we also remove Freeefont?

File: .github/workflows/ubuntu.yml [hunk 2]
```diff
@@ -26,7 +26,7 @@ jobs:
             software-properties-common gcc ruby ruby-dev libffi-dev make libxml2-dev libxslt1-dev
 
           # Install fonts
-          sudo apt-get -y install fonts-freefont-ttf fonts-droid-fallback
+          sudo apt-get -y install fonts-freefont-ttf
```
Comment (ronaldtse) [inline .github/workflows/ubuntu.yml]: Remove Freefont?

File: README.adoc [hunk 3]
```diff
@@ -120,7 +92,7 @@ sudo apt-get -y install gettext-base cabextract wget
 sudo apt-get -y install software-properties-common
 
 # Install fonts
-sudo apt-get -y install fonts-freefont-ttf fonts-nanum fonts-droid-fallback
+sudo apt-get -y install fonts-freefont-ttf
```
Comment (ronaldtse) [inline README.adoc]: Remove Freefont?

File: .github/workflows/ubuntu.yml (hunk 2 above)
Comment (Intelligent2013) [reply .github/workflows/ubuntu.yml]: Yes, you're right. It should be removed. I've fixed it.

File: README.adoc (hunk 3 above)
Comment (Intelligent2013) [reply README.adoc]: Fixed.

File: .github/workflows/debian.yml (hunk 1 above)
Comment (Intelligent2013) [reply .github/workflows/debian.yml]: Fixed.

**Pull request merged**